docs/community/team.md

```diff
@@ -17,10 +17,34 @@ napari is a consensus-based community project. Anyone with an interest in the pr
 - [Talley Lambert](https://github.com/napari/napari/commits?author=tlambert03) - [@tlambert03](https://github.com/tlambert03)
 - [Ziyang Liu](https://github.com/napari/napari/commits?author=ziyangczi) - [@ziyangczi](https://github.com/ziyangczi)
 
+<sub>***SC** on the napari [steering council](https://napari.org/community/governance.html#steering-council).</sub>
 
 ## Emeritus Core Developers
 
 - [Shannon Axelrod](https://github.com/napari/napari/commits?author=shanaxel42) - [@shanaxel42](https://github.com/shanaxel42)
 
 
-<sub>***SC** currently on the napari [steering council](https://napari.org/community/governance.html#steering-council).</sub>
+## Project History
+
+The napari project began in the spring of 2018. It was born out of the shared need for fast n-dimensional image viewing in Python by project co-founders Loïc Royer, from the Chan Zuckerberg Biohub (CZ Biohub), and Juan Nunez-Iglesias, from Monash University. Since then it has grown rapidly to now also provide a graphical user interface to a plugin ecosystem of image analysis tools for scientists to use in their daily work.
+
+Named after the tiny village of Napari, in the Republic of Kiribati in the Pacifc Ocean, at the geographic midpoint between Loïc's home of San Francisco and Juan’s home of Melbourne, the napari project has, from its inception, been a global community effort.
+
+Early development on the project was led by Kira Evans, first as an intern with Loïc at CZ Biohub, then later as a software engineer at the Chan Zuckerberg Initiative (CZI). By the fall of 2018, Nicholas Sofroniew (CZI) joined in, accelerating both the organization and management of the growing team and the development of core features.
+
+You can read a full history of the project’s founding and early development in this [blog post](https://ilovesymposia.com/2019/10/24/introducing-napari-a-fast-n-dimensional-image-viewer-in-python/) from Juan.
+
+Since then, napari has grown to over 80 direct contributors. Talley Lambert, from Harvard Medical School, began contributing around the time Juan wrote about napari’s founding, and has been a critical driver of napari's development ever since. We have core developers located in three continents and three biweekly community calls spread across the day to try to accommodate users and contributors in any time zone.
```

Comment:
```suggestion
Over time, napari has grown to over 80 direct contributors. Talley Lambert, from Harvard Medical School, began contributing around the time of Juan's blog post, and has been a critical driver of napari's development ever since. We have core developers located in three continents and three biweekly community calls spread across the day to try to accommodate users and contributors in any time zone.
```

We've already used "since then" so I think it would be nice to switch it up a bit here.